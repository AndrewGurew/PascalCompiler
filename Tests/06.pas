program test;
var a:Integer;
