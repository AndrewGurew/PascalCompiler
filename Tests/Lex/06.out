Position            Type                     Text                Value               
-------------------------------------------------------------------------------------
(1,1)               Programm                 program             program             
(1,9)               Id                       test                test                
(1,13)              Semicolon                ;                   ;                   
(2,2)               Var                      var                 var                 
(2,6)               Id                       a                   a                   
(2,7)               Colon                    :                   :                   
(2,8)               Integer                  Integer             Integer             
(2,15)              Semicolon                ;                   ;                   
(3,15)              End of file              EndOfFile           EndOfFile           
